### DISCUSSION TITLE
How to autoscale in Milvus kubernetes

### DISCUSSION THREAD

**Discussion by DimasAdinugroho** [milvus-io/milvus#27916] [Q&A and General discussion]
How do we do the autoscaling on Milvus based on the resource utilization, let's say using HPA ? All I see in the docs is the manual method.

**Comment by yhmo** [comment]
So far as I know, milvus doesn't support autoscaling. Wait @LoveEachDay to confirm.

**Comment by filip-halt** [comment]
At the moment we have not implemented autoscaling support. While the metrics are there it would require some work from the user to achieve this.

**Reply by DimasAdinugroho** [reply]
is there any workaround? I tried to use `HPA` but it seems that the `Deployment` is controlled by `Milvus Operator`
